docs/book/src/development/releasing.md

```diff
@@ -4,25 +4,26 @@
 
 1. Make sure your repo is clean by git's standards
 2. If this is a new minor release, create a new release branch and push to github, otherwise switch to it, for example `release-0.2`
-3. Run `make release-notes` to gather changes since the last revision. If you need to specify a specific tag to look for changes
```

Comment:
Do we want to remove this?

Reply:
Yup. `make release` will output it in out/CHANGELOG.md, and have amended this doc to reference it.